Sources/Sentry/SentryUIEventTracker.m

```diff
@@ -46,21 +46,27 @@ - (void)start
                     return;
                 }
 
+                NSString *operation = @"ui.action";
+                if (event
+                    && (event.type == UIEventTypePresses || event.type == UIEventTypeTouches)) {
+                    operation = @"ui.action.click";
+                }
+
                 [span finish];
 
                 Class targetClass = [target class];
                 NSString *transactionName;
 
                 if (targetClass) {
                     transactionName = [NSString
-                        stringWithFormat:@"[%@ %@]", NSStringFromClass(targetClass), action];
+                        stringWithFormat:@"%@.%@", NSStringFromClass(targetClass), action];
```

Comment:
l: Depending on the action, it may have the following format: `firstArgument:SecondArgument:` 
And the name would be `ViewController.firstArgument:SecondArgument:` which is not Swift friendly I guess.

But there is no perfect solution here, so, Im fine if you want to keep this format.

Reply:
The `#selector` syntax in swift would be

- `SomeClass.myMethod(first:second:third:)`
- `SomeClass.myMethod(first:)`
- `SomeClass.myMethod`

What do you think about that, @brustolin?

Reply:
This would be the best solution, a little trickier though.